app/js/actions/labOrders/fetchLabOrders.js

```diff
@@ -0,0 +1,36 @@
+import 'babel-polyfill';
+import axiosInstance from '../../config';
```

Comment:
How about you import the babel-polyfill in the entry point of this client codebase so that it can be used in other parts of the codebase without re-importing.

Reply:
This has been fixed in another Pull request.

Reply:
Owk.